vendor/gradle-plugin/build.gradle

```diff
@@ -88,39 +94,36 @@ allprojects {
 }
 
 
-def computeCompileSdkVersion = { -> project.hasProperty("compileSdk") ? compileSdk : 28 }
-def computeBuildToolsVersion = { ->
-    project.hasProperty("buildToolsVersion") ? buildToolsVersion : "28.0.3"
-}
-
 android {
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
Do we need to set the target to 1.8 with gradle7+? Officially recommended jdk is 11, so this might need to match that?

Reply:
we should be able to use more but was not tested https://kotlinlang.org/docs/faq.html#which-versions-of-jvm-does-kotlin-target
let s make it an ext property so we can test easily?